src/DynamoCoreWpf/ViewModels/Core/DynamoViewModelDelegateCommands.cs

```diff
@@ -153,5 +154,6 @@ private void InitializeDelegateCommands()
         public DelegateCommand DumpLibraryToXmlCommand { get; set; }
         public DelegateCommand ShowGalleryCommand { get; set; }
         public DelegateCommand CloseGalleryCommand { get; set; }
+        public DelegateCommand ShowNewDesignOptionDialogCommand { get; set; }
```

Comment:
Let's get rid of the `DesignOption` usage here - remember be ruthlessly consistent.  It really helps to keep the code maintainable.  :)